users/views/identity.py

```diff
@@ -64,10 +66,13 @@ def serve_actor(self, identity):
         # If this not a local actor, redirect to their canonical URI
         if not identity.local:
             return redirect(identity.actor_uri)
-        return JsonResponse(
+        r = JsonResponse(
             canonicalise(identity.to_ap(), include_security=True),
             content_type="application/activity+json",
         )
+        if identity.deleted and tz.now() - identity.deleted > tz.timedelta(days=3):
```

Comment:
Can you not inline this formula? It's not complicated, but it is subtle and it took me a moment to figure out what it was doing and why.